pypeit/core/procimg.py

```diff
@@ -606,6 +671,38 @@ def trim_frame(frame, mask):
     return frame[np.invert(np.all(mask,axis=1)),:][:,np.invert(np.all(mask,axis=0))]
 
 
+def init_process_steps(bias, proc_par):
```

Comment:
It appears this function is only used by the new images classes you created so consider moving it to those modules.

Reply:
If you ctrl-click on this method in PyCharm
you can see all the places that a method is called.
I encourage you to do that.